packages/mds-compliance-service/repository/entities/compliance-snapshot-kafka-failure.ts

```diff
@@ -39,14 +40,14 @@ export interface ComplianceSnapshotKafkaFailureEntityModel extends IdentityColum
   timestamp: ComplianceSnapshotKafkaFailureDomainModel['timestamp']
 }
 
-@Entity('compliance_snapshots')
+@Entity('compliance_snapshot_failures')
 export class ComplianceSnapshotKafkaFailureEntity
   extends IdentityColumn(RecordedColumn(class {}))
   implements ComplianceSnapshotKafkaFailureEntityModel
 {
   @Column('uuid', { primary: true })
-  compliance_snapshot_id: ComplianceSnapshotKafkaFailureEntityModel['compliance_snapshot_id']
+  compliance_snapshot_id: UUID
```

Comment:
Probably not an issue here, but the cursor pagination library we use doesn't like mapped types in the entity definition. The reflection library thinks they are objects instead of primitive types.